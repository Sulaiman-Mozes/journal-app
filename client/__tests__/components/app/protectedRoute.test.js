import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import ProtectedRoutes from '../../../components/app/protectRoute';

describe('Provider and ProtectedRoutes', () => {
  const protectedRoutes = shallow(<ProtectedRoutes component={jest.fn()} />);

  it('renders <Provider/> correctly', () => {
    expect(protectedRoutes).toMatchSnapshot();
  });
});
