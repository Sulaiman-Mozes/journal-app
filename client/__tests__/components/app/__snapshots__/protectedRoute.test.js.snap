// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider and ProtectedRoutes renders <Provider/> correctly 1`] = `
<Route
  render={[Function]}
/>
`;
